docs/DeveloperGuide.md

```diff
@@ -149,35 +148,42 @@ How the parsing works:
 * does not depend on any of the other three components (as the `Model` represents data entities of the domain, they should make sense on their own without depending on other components)
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Position` list in the `AddressBook`, which `Member` references. This allows `AddressBook` to only require one `Position` object per unique POSITION, instead of each `Member` needing their own set of `Position` objects.<br>
+Within the `Model` component,
+* `Module` is the superclass of `Member`, `Event` and `Task`. `Module` has a `Name` field as the common attribute of its subclasses.
+* `Event` has a `Name` field, `EventDate` field, and a field of a `HashMap<Member, Boolean>` to serve as a participant list with attendance.
+* `Member` has its original fields from AB3 and an additional field of a `TaskList` to store all tasks assigned to the member.
+* `Task` has a `Name` field, `TaskDeadline` field, and a `Boolean` field to represent the completion status of the task.
 
+An alternative (arguably, a more OOP) model design is given below,
 
-<img src="images/BetterModelClassDiagram.png" width="450" />
+<img src="images/AlternativeModelClassDiagram.png" width="600" />
 
-</div>
-
-[comment]: <> (TODO: CHIXU)
-#### Current Implementations
+* `AddressBook` also stores all `Task` objects (which are contained in a `UniqueTaskList` object).
+* Each `Member` object references `Task` object in the `UniqueTaskList`, instead of needing its own list of tasks.
+However, this design has some issues in storage,
+* Since Ailurus uses json files to store user data, json format members should at least store the unique identifier of each task the member had.
+In our implementation, the unique identifier is `Name` and `TaskDeadline`, which are actually the main part of a `Task`.
+* At the same time, json format member need to store the completion status of each referencing task.
+* In all, implementing the storage of this member-task relation by using json file is likely to incur redundancy and error-prone,
+so we desided to use a easier implementation, which is the current one.
 
-The model that we implemented currently has `Event`, `Task` and `Member`. `Member` has a field with `TaskList` which contains
-`Task` belonging to the `Member`. `Event` has a `Name` field, `EventDate` field, and a field of a HashMap<Member, Boolean>
-to serve as a participant list with attendance. Event and Member both extend from the abstract class `Module` to reduce class duplication.
+A better implementation of the alternative design may involve using database management system like PostgreSQL,
+a proposed entity relationship model diagram for the member-task relation is given below:
 
-The `Task` model we implemented currently has a task name and a state that represents it is done or not.
+<img src="images/ER.png" width="600"/>
```

Comment:
To change to a link instead of image as ER diagram not really learned in this module

Reply:
just this